### PR TITLE
Allison fain

### PR DESCRIPTION
**Pull request by allisonfain08**
working on adding connections from storyboard to cocoa touch class. have bug w system rn but will have working by later today. in meantime, go ham with the storyboard ui edits.

### CODE REVIEW

File: tryingsomethingyetagain/team2120/team2120.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+				2A106C0828EF135900B93B9B /* Debug */,
+				2A106C0928EF135900B93B9B /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
+	};
+	rootObject = 2A106BF128EF135800B93B9B /* Project object */;
+}
```
Comment (MijaKennedy) [inline tryingsomethingyetagain/team2120/team2120.xcodeproj/project.pbxproj:378]: A bit long here to give detailed feedback. Assuming this runs fine, it looks fine to me.

File: tryingsomethingyetagain/team2120/team2120/QuizViewController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//
+//  QuizViewController.swift
+//  team2120
+//
+//  Created by Allison Fain on 10/6/22.
+//
+
+import UIKit
+
+class QuizViewController: UIViewController {
```
Comment (MijaKennedy) [inline tryingsomethingyetagain/team2120/team2120/QuizViewController.swift:10]: Do we only need one quiz view controller? Are we certain we are only doing one quiz format?

File: tryingsomethingyetagain/team2120/team2120/team2120App.swift [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+//  team2120App.swift
+//  team2120
+//
+//  Created by Allison Fain on 10/6/22.
+//
+
+import SwiftUI
+
+@main
+struct team2120App: App {
```
Comment (MijaKennedy) [inline tryingsomethingyetagain/team2120/team2120/team2120App.swift:11]: overall good setup for needed controllers/views.

Comment (MijaKennedy) [reply tryingsomethingyetagain/team2120/team2120/team2120App.swift:11]: should be a good baseline for building the app

**Review by MijaKennedy — COMMENTED** [review]
Overall looks good, some files too long to parse well but setup looked sufficient for continued progress on the app for sprint 3

**Review by Yugenero — APPROVED** [review]

**Review by ShonjoyBhattacharyya — COMMENTED** [review]
Changes look good to me! It's a great starting point for this app

**Review by olivia-mauger — COMMENTED** [review]
This looks good - we will need to adjust our iteration plan and relay to client that we are only working in ios right now.

**Review by Yugenero — APPROVED** [review]
Looks good to me.

File: tryingsomethingyetagain/team2120/team2120/team2120App.swift (hunk 3 above)
Comment (allisonfain08) [reply tryingsomethingyetagain/team2120/team2120/team2120App.swift:11]: sounds good!

**Review by sreya101 — APPROVED** [review]
Looks good to me!

**Pull request merged**